Resources/config/services.xml

```diff
@@ -46,7 +46,16 @@
         <service id="cmf_seo.error.suggestion_provider.controller" class="%cmf_seo.error.suggestion_provider.controller.class%">
             <argument type="service" id="twig" />
             <argument>%kernel.debug%</argument>
+            <argument type="service" id="cmf_seo.error.exclusion_matcher" />
+            <argument>%cmf_seo.error.configurations%</argument>
         </service>
+
+        <service id="cmf_seo.error.request_matcher"
+                 class="Symfony\Component\HttpFoundation\RequestMatcher"
+                 public="false"
```

Comment:
you probably would need to un-abstract this with the decorator however, not exactly sure how that works